anyway/constants/backend_constants.py

```diff
@@ -0,0 +1,109 @@
+from enum import Enum
+from typing import List
+from flask_babel import _
+
+
+class BackEndConstants(object):
```

Comment:
As we are doing a big change, I do not see value in having the constants in this file inside a class.